.github/workflows/github_actions.yml

```diff
@@ -0,0 +1,55 @@
+name: Coverity scan
+
+on:
+  push:
+    branches: [develop]
+
+jobs:
+  coverity-scan:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout code
+        uses: actions/checkout@v2
+      - name: install dependencies
+        run: |
```

Comment:
Can you move the following commands into a dedicated shell script? Something like [.ci/setup.sh](https://github.com/DLTcollab/sse2neon/tree/master/.ci).